Docs/Help/Build-Bicep.md

```diff
@@ -49,7 +52,7 @@ Build-Bicep -Path vnet.bicep
 
 ### Example 2: Compile single bicep file and specify the output directory
 ```powershell
-Build-Bicep -Path 'c:\bicep\modules\vnet.bicep' - -OutputDirectory 'c:\armtemplates\vnet.bicep'
+Build-Bicep -Path 'c:\bicep\modules\vnet.bicep' -OutputDirectory 'c:\armtemplates\vnet.bicep'
```

Comment:
While you´re at it. Can you fix this line. It should be:
 `Build-Bicep -Path 'c:\bicep\modules\vnet.bicep' -OutputDirectory 'c:\armtemplates\'`